### DISCUSSION TITLE
GEMINI examples that include Data  quality

### DISCUSSION THREAD

**Discussion by PeterParslow** [agiorguk/gemini#145] [General]
Hi,
I am corresponding with someone working on their first GEMINI records and she asks about the Data quality element (https://agiorguk.github.io/gemini/1062-gemini-datasets-and-data-series.html#52). I know I wrote the guidance, but does anyone have examples I can point her to that actually use this element?

Peter

@nmtoken  @archaeogeek @6footdestiny

**Comment by archaeogeek** [comment]
We have some xml snippets at https://github.com/AstunTechnology/geonetwork-snippets/tree/main/dataquality for the main dataquality elements if that's any help?

**Reply by PeterParslow** [reply]
Both links in this comment are the same; is that intentional?

I'll point my correspondent at them. She's using the ArcGIS metadata editor, but at least she'll have a clear idea what she's trying to get it to create!

**Comment by archaeogeek** [comment]
edited- just a typo
